tests/Integration/SeasonControllerTest.php

```diff
@@ -190,4 +190,39 @@ public function testShouldNotIncludeContinuingItemsByDefault()
         $this->assertIsArray($content["data"]);
         $this->assertCount(2, $content["data"]);
     }
+
+    public function testShouldNotIncludeNewlyStartedSeasonOfAnimeInPreviousSeasons()
+    {
+        Carbon::setTestNow(Carbon::parse("2024-10-26"));
+        $f = Anime::factory(1);
+        $startDate = "2024-10-02";
+        $carbonStartDate = Carbon::parse($startDate);
+        $state = $f->serializeStateDefinition([
+            "aired" => new CarbonDateRange($carbonStartDate, null)
+        ]);
+        $state["aired"]["string"] = "Oct 2, 2024 to ?";
+        $state["airing"] = true;
+        $state["status"] = "Currently Airing";
+        $state["premiered"] = "Fall 2024";
+        $state["mal_id"] = 54857;
+        $state["title"] = "Re:Zero kara Hajimeru Isekai Seikatsu 3rd Season";
```

Comment:
Wouldn't this test pass in any case in the future?

Reply:
Not sure what you are asking. Are you asking about the mocks in the test or in overall?

Reply:
I meant overall, missed that this would be the mock tests. In that case the data would be static so it sounds A-OK.